plugins/performance-lab/includes/admin/load.php

```diff
@@ -465,7 +465,7 @@ function perflab_get_plugin_settings_url( string $plugin_slug ): ?string {
 		return null;
 	}
 	$href = $p->get_attribute( 'href' );
-	if ( $href && is_string( $href ) ) {
+	if ( is_string( $href ) && '' !== $href ) {
```

Comment:
I could go either way here, but for consistency...
```suggestion
	if ( null !== $href && '' !== $href ) {
```

Reply:
This should stay as is because if `<a href>` is encountered, then `$href` here would be `true` which would be a type error since the function returns a `string` or `null`.